Test/ServerTelemetryChannel.Test/Shared.Tests/Helpers/StubTelemetryBuffer.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel.Helpers
+{
+    using System;
+    using System.Collections.Generic;
+    using Microsoft.ApplicationInsights.Channel;
+    using Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel.Implementation;
+    using System.Threading.Tasks;
+
+    internal class StubTelemetryBuffer : Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel.Implementation.TelemetryBuffer
```

Comment:
Please doc comment all new code (and any existing code you feel like cleaning up while you are editing it).